docs/inputs/conditional-input.md

```diff
@@ -8,91 +8,137 @@ sidebar_position: 5
 | :trophy: This is a [premium](https://mecsimcalc.com/pricing) feature |
 | -------------------------------------------------------------------- |
 
-The following provides a detailed description about the Conditional Input feature
+The following provides a detailed description about the **Conditional Input** feature
+
+## Why use it?
+
+The **Conditional Input** feature allows you to create more complex and dynamic user interfaces. You can now hide or disable an **Input** based on the value of other **Inputs**!
```

Comment:
should it be **other Inputs** instead of just **Inputs**, and whenever there is a reference to the input we are checking it would be for example: "we check the **other Input(s)**"